pyrometheus/chem_expr.py

```diff
@@ -164,8 +167,61 @@ def _zeros_like(argument):
 
 # }}}
 
-# {{{ Equilibrium constants
 
+# {{{ Transport polynomials & mixture rules
+
+def transport_polynomial_expr(c, n, t):
```

Comment:
Many of your docstrings look quite off in terms of formatting. Please render the documentation via sphinx to make sure they look OK.

Reply:
@tulioricci , fwiw the doc _build_ itself is OK.  The warning is due to not having the example mechanism available when it builds the doc.  This line:

https://github.com/tulioricci/pyrometheus/blob/a49204befbe2c22deadcba1e3e412d9bf40b114d/.github/workflows/ci.yml#L88

in the CI config builds that example before running the doc build, which is why CI does not encounter this issue.